README.md

```diff
@@ -225,6 +225,18 @@ In the above example, `Dish__c` is a Salesforce object type which references the
 
   You may want to consult [the ActiveRecord documentation](http://apidock.com/rails/ActiveRecord/Associations/ClassMethods) for your specific use case.
 
+### Add an external ID to your Salesforce objects
+
+If your application creates any objects that you want/need to propagate to Salesforce, you'll need to expose an external ID field named `SynchronizationId__c` on the Salesforce object.
```

Comment:
Might be worth adding something like "This field is used to prevent accidental creation of duplicate records in Salesforce blah blah blah copy the good explanations you already wrote elsewhere"